sphinx/builders/__init__.py

```diff
@@ -558,8 +558,10 @@ def write(
         with progress_message(__('preparing documents')):
             self.prepare_writing(docnames)
 
-        with progress_message(__('copying assets')):
```

Comment:
This will not be sufficient since the `progress_message` context has a specific exit handler (which handles exceptions that can include skip reasons).

Instead, the patch should fix `progress_message` so that the latter takes a constructor argument `nonl=True/False` and use that value when entering the context (upon exiting the context, nothing should be changed I think?).

Reply:
Is this OK?

```
copying assets...
copying downloadable files... [100%] includes/tzinfo_examples.py
copying static files... done
copying extra files... done
copying assets... done
```

Note that "copying assets..." shown twice.

Reply:
Maybe we could have something like "copying assets: done" ? with a colon so that we know that the message is for the "copying asset" instead of "..."?